coffee-jpa/src/main/java/hu/icellmobilsoft/coffee/jpa/sql/batch/BatchService.java

```diff
@@ -82,6 +79,9 @@
 import hu.icellmobilsoft.coffee.jpa.sql.entity.EntityHelper;
 import hu.icellmobilsoft.coffee.tool.utils.string.RandomUtil;
 
+import jakarta.inject.Inject;
+import jakarta.persistence.EntityManager;
+
```

Comment:
import sorrend maradhatna java;javax;jakarta ...

Reply:
a java-dev-tools readme-nek megfelelően most ezt állítottam be: import static all other imports; blank line; import java.*; blank line; import javax.*; blank line; import jakarta.*; blank line; import org.*; blank line; import all other

Reply:
akkor a dev-toolsban megsem lett jo a formatter? ott igy formaz pl:
import java....; 
import hu...
import jakarta....

Reply:
de amúgy most itt is úgy van, csak bejön az org a java.... és a hu.... közé

Reply:
meglett az eltérés oka, most valamiért máshol kell megadni IntelliJ-ben az import sorrendet mint eddig